test/executor.go

```diff
@@ -15,7 +15,7 @@ type Executor struct {
 // NewExecutor returns a new instance of Executor.
 // The executor always matches the hostname of the first cluster node.
 func NewExecutor(holder *pilosa.Holder, cluster *pilosa.Cluster) *Executor {
-	executor := pilosa.NewExecutor(nil)
+	executor := pilosa.NewExecutor(pilosa.GetHTTPClient(nil))
```

Comment:
would it be better to use a default client defined at the package level here, so that we aren't newing up lots of new clients while running tests?